diff -durN src.orig/racer/build.sh src/racer/build.sh
--- src.orig/racer/build.sh	Wed Aug  1 09:29:31 2001
+++ src/racer/build.sh	Sun Aug  5 14:48:18 2001
@@ -17,7 +17,8 @@
 # If you encounter other problems consult the racer linux FAQ at 
 # http://www.schuerkamp.de/racer_linux_faq.html
 
-export DEFCFLAGS="-I$HOME/src/racer/include -I$HOME/src/racer/include/linux"
+export BASEDIR=`pwd`
+export DEFCFLAGS="-I$BASEDIR/include -I$BASEDIR/include/linux -I/usr/X11R6/include"
 export DEFLIBS="-L../lib -lD3 -lraceru -L lib -lracer -lD3 -lQ -L/usr/X11R6/lib -lGL -lGLU -lm"
 
 echo cleaning up....
diff -durN src.orig/racer/libs/d3/makefile src/racer/libs/d3/makefile
--- src.orig/racer/libs/d3/makefile	Tue Jul 31 19:04:27 2001
+++ src/racer/libs/d3/makefile	Sun Aug  5 14:48:18 2001
@@ -4,7 +4,7 @@
 
 #include $(ROOT)/usr/include/make/commondefs
 
-TARGET=$(HOME)/src/racer/racer/lib/libD3.a
+TARGET=$(BASEDIR)/racer/lib/libD3.a
 
 DFILES=dglobal.o dscene.o dcull_sl.o dculler.o dbatch.o dfps.o\
  dobb.o\
@@ -21,7 +21,7 @@
 # Old IRIX6.3 should use -n32 to generate new-style 32-bit
 #CFLAGS=-woff 3262,1356,1174 -DDEBUG -O2 -I$(HOME)/include
 #CFLAGS=$(DEFCFLAGS) -DDEBUG -O2 -woff 1174 -mips3
-CFLAGS=$(DEFCFLAGS) -DDEBUG -O2 -DLINUX -I$(HOME)/src/racer/racer
+CFLAGS=$(DEFCFLAGS) -DDEBUG $(OPTFLAGS) -DLINUX -I$(BASEDIR)/racer
 
 AR=ar
 ARFLAGS=r
@@ -33,7 +33,6 @@
 
 default: $(TARGET)
 	@echo $(TARGET) is up to date.
-	@beep
 
 $(TARGET): $(FILES)
 	$(AR) $(ARFLAGS) $(TARGET) $(FILES)
diff -durN src.orig/racer/libs/qlib/makefile src/racer/libs/qlib/makefile
--- src.orig/racer/libs/qlib/makefile	Thu Aug  2 12:49:53 2001
+++ src/racer/libs/qlib/makefile	Sun Aug  5 14:48:18 2001
@@ -4,7 +4,7 @@
 
 #include $(ROOT)/usr/include/make/commondefs
 
-TARGET=$(HOME)/src/racer/racer/lib/libQ.a
+TARGET=$(BASEDIR)/racer/lib/libQ.a
 
 GFXFILES=qtext.o qmoviebob.o qbob.o qbgr.o qshape.o qgel.o\
  qfxman.o qvideofx.o qfx.o qfxlist.o\
@@ -88,9 +88,9 @@
 # Old IRIX6.3 should use -n32 to generate new-style 32-bit
 #CFLAGS=-woff 3262,1356,1174 -DDEBUG -O2 -I$(HOME)/include
 # SGI IRIX flags
-CFLAGS=$(DEFCFLAGS) -DDEBUG -O2 -mips3
+#CFLAGS=$(DEFCFLAGS) -DDEBUG -O2 -mips3
 # Linux flags
-CFLAGS=$(DEFCFLAGS) -DDEBUG -DLINUX -DQ_USE_SDL
+CFLAGS=$(DEFCFLAGS) -DDEBUG $(OPTFLAGS) -DLINUX -DQ_USE_SDL
 
 #ARFLAGS=crl
 #AR=CC
@@ -105,7 +105,6 @@
 
 default: $(TARGET)
 	@echo QLib is up to date.
-	@beep
 
 $(TARGET): $(QFILES)
 	$(AR) $(ARFLAGS) $(TARGET) $(QFILES)
diff -durN src.orig/racer/racer/lib/makefile src/racer/racer/lib/makefile
--- src.orig/racer/racer/lib/makefile	Fri Aug  3 13:27:23 2001
+++ src/racer/racer/lib/makefile	Sun Aug  5 14:48:47 2001
@@ -15,7 +15,7 @@
  rdriver.o rstats.o\
  rprofile.o rflat.o
 CC=g++
-CFLAGS=$(DEFCFLAGS) -DDEBUG -O2 -DQ_USE_SDL
+CFLAGS=$(DEFCFLAGS) -DDEBUG $(OPTFLAGS) -DQ_USE_SDL
 
 AR=ar
 ARFLAGS=r
@@ -27,11 +27,10 @@
 
 default: $(TARGET)
 	@echo $(TARGET) is up to date.
-	@beep
 
 $(TARGET): $(FILES)
 	$(AR) $(ARFLAGS) $(TARGET) $(FILES)
-	cp $(TARGET) $(HOME)/src/racer/racer/lib
+	cp $(TARGET) $(BASEDIR)/racer/lib
 
 depend:
 	makedepend -I$(HOME)/include *.cpp
diff -durN src.orig/racer/racer/libu/makefile src/racer/racer/libu/makefile
--- src.orig/racer/racer/libu/makefile	Tue Jul 31 19:15:43 2001
+++ src/racer/racer/libu/makefile	Sun Aug  5 14:48:18 2001
@@ -8,7 +8,7 @@
 
 FILES=track.o ui.o
 CC=gcc
-CFLAGS=$(DEFCFLAGS) -DDEBUG -O2
+CFLAGS=$(DEFCFLAGS) -DDEBUG $(OPTFLAGS)
 
 AR=ar
 ARFLAGS=r
@@ -20,7 +20,6 @@
 
 default: $(TARGET)
 	@echo $(TARGET) is up to date.
-	@beep
 
 $(TARGET): $(FILES)
 	$(AR) $(ARFLAGS) $(TARGET) $(FILES)
diff -durN src.orig/racer/racer/makefile src/racer/racer/makefile
--- src.orig/racer/racer/makefile	Fri Aug  3 13:57:10 2001
+++ src/racer/racer/makefile	Sun Aug  5 14:48:18 2001
@@ -27,7 +27,7 @@
 CC=g++
 
 #CFLAGS=$(DEFCFLAGS) -I$(HOME)/include -DDEBUG -pch -woff 1174 -O2
-CFLAGS=$(DEFCFLAGS) -I$(HOME)/include -O2 -DLINUX -DQ_USE_SDL
+CFLAGS=$(DEFCFLAGS) -I$(HOME)/include $(OPTFLAGS) -DLINUX -DQ_USE_SDL
 #LFLAGS=-L$(HOME)/lib -lD3 -lraceru -lracer $(DEFLIBS) 
 # Linux uses SDL
 LFLAGS=-L$(HOME)/lib -lD3 -lraceru -lracer $(DEFLIBS) -lSDL -lSDL_mixer
@@ -41,7 +41,7 @@
 
 targets: $(PRJ)
 
-$(PRJ): $(OBJS) $(HOME)/src/racer/racer/lib/libracer.a
+$(PRJ): $(OBJS) $(BASEDIR)/racer/lib/libracer.a
 	$(CC) -o $(PRJ) $(OBJS) $(LFLAGS)
 
 #include $(COMMONRULES)
